snafu/process.py

```diff
@@ -223,13 +76,41 @@ def get_process_sample(
     tries: int = 0
     tries_plural: str = ""
 
```

Comment:
What is the purpose of `capture_output`? Is this for backwards compatibility?

Reply:
Yeah exactly. The kwarg isn't available in 3.6, so it needs to be removed before being given to `subprocess.run`.